docs/source/commandline.md

```diff
@@ -184,6 +184,327 @@ Create a basic [solid](https://www.solidjs.com/) app.
 | [webapp](https://atmospherejs.com/meteor/webapp)                                                     |                     |          |          |      X      |           |            |         |            |              |               |           |
 | [react-meteor-data](https://atmospherejs.com/meteor/react-meteor-data)                               |          X          |          |          |             |           |            |         |            |      X       |       X       |           |
 
+<h2 id="meteorgenerate"> meteor generate </h2>
+
+``meteor generate`` is a command for generating scaffolds for your current project. when ran without arguments, it will ask
+you what is the name of the model you want to generate, if you do want methods for your api and publications. It can be
+used as a command line only operation as well.
+
+running 
+```bash
+meteor generate customer
+
+```
+
+it will generate the following code in ``/imports/api``
+![Screenshot 2022-11-09 at 11 28 29](https://user-images.githubusercontent.com/70247653/200856551-71c100f5-8714-4b34-9678-4f08780dcc8b.png)
+
+that will have the following code:
+
+
+<h3 id="meteorgenerate-collection.js">collection.js</h3>
+
+```js
+
+ import { Mongo } from 'meteor/mongo';
+
+export const CustomerCollection = new Mongo.Collection('customer');
+
+```
+
+
+
+<h3 id="meteorgenerate-methods.js">methods.js</h3>
+
+```js
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+import { CustomerCollection } from './collection';
+
+export function create(data) {
+  return CustomerCollection.insertAsync({ ...data });
+}
+
+export function update(_id, data) {
+  check(_id, String);
+  return CustomerCollection.updateAsync(_id, { ...data });
+}
+
+export function remove(_id) {
+  check(_id, String);
+  return CustomerCollection.removeAsync(_id);
+}
+
+export function findById(_id) {
+  check(_id, String);
+  return CustomerCollection.findOneAsync(_id);
+}
+
+Meteor.methods({
+  'Customer.create': create,
+  'Customer.update': update,
+  'Customer.remove': remove,
+  'Customer.find': findById
+});
+
+```
+
+
+
+
+
+<h3 id="meteorgenerate-publication.js">publication.js</h3>
+
+```js
+
+import { Meteor } from 'meteor/meteor';
+import { CustomerCollection } from './collection';
+
```

Comment:
This generated publication might be a little confusing as the `customer` collection might not have a `userId`.

Reply:
you think is best to just leave the ``all$$CollectionName$$`` ?

Reply:
I guess so. But if you want to keep it, no problem.